src/audio/multiband_drc/multiband_drc.c

```diff
@@ -32,6 +31,8 @@
 #include <errno.h>
 #include <stddef.h>
 #include <stdint.h>
+
+#include "../drc/drc_algorithm.h"
```

Comment:
as @marc-hb also has pointed out elsewhere, such includes aren't very pretty, would be better to keep shared between modules headers in an "include" directory

Reply:
This isn't a fully clearcut case either. Here multiband_drc is directly building on top of drc (and also uses crossover).

I think we should also ensure this type of cross-module resuse is easy to do and code stays maintainable. 

@lgirdwood Any input, should we keep such reused headers (like drc_algorithm.h) in common code?

I think I'd personally still code with the approach in this PR. The modules are deeply dependent on each other, so having a "../drc/drc_algorithm.h" makes sense. Having alone drc_algorithm.h in sof/audio/ seems harder to understand ("why is this header here but not others?").

Reply:
I don't have a huge problem with this approach, just looking for the best one. As alternatives we could either
1. have an include directory under src/audio/include for such shared headers or
2. add src/audio to the list of include paths

I think I'd find (2) preferable

Reply:
Seppo suggested move the common part to math folder in a separate PR, I will do it once those PRs get approval and merged.

Reply:
> The modules are deeply dependent on each other, so having a "../drc/drc_algorithm.h" makes sense. 

I measured how frequent are these "dot-dot" includes across all the default `zephyr/west.yml` repos. It's about 1.5%. So it's not "exotic" but it's definitely not "mainstream" either. As I just mentioned in #8630, there is `zephyr_library_include_directories()` and friends.

```
$ west grep '^[[:blank:]]*#[[:blank:]]*include[[:blank:]]*[<"]\.\.' | wc -l
2914
$ west grep '^[[:blank:]]*#[[:blank:]]*include[[:blank:]]*[<"][[:alnum:]]' | wc -l
228292
```

Reply:
how  about non-zephyr cases? i.e: CONFIG_ZEPHYR_NATIVE_DRIVERS is not defined, will other build also support zephyr syntax?